### ISSUE TITLE
Need new guide

### ISSUE THREAD

**Issue by chassain** [kapsolas/MMM-Instagram#22]
Hello,

It will be nice to have a new guide because new client registration and permission review on Instagram API platform are discontinued in favor of the Instagram Basic Display API.
I'm noob but i would like to try this nice module..(i have loading after 2 days tries)
Thanks

**Comment by payno4** [comment]
if there would be a new guide that would be nice. I want to try this module out, it seems to be a really cool one.

**Comment by Bernard-h** [comment]
git clone command during install asks for github.com username / and password . I have no idea how to get it to install

**Comment by Bernard-h** [comment]
Agreed , would be nice to have a good guide on the new API side of things. Or perhaps the link of a current which I cant seem to find. PixelSolutions isnt working for me but support would drop early 2020... :(

**Comment by keant1** [comment]
Agreed, I'd definitely love to add this to my Magic Mirror as well
